flink-table/flink-table-planner-blink/src/main/scala/org/apache/flink/table/api/TableConfig.scala

```diff
@@ -102,17 +97,19 @@ class TableConfig {
   private var conf = GlobalConfiguration.loadConfiguration()
 
   /**
-   * Sets the timezone for date/time/timestamp conversions.
-   */
-  def setTimeZone(timeZone: TimeZone): Unit = {
-    require(timeZone != null, "timeZone must not be null.")
-    this.timeZone = timeZone
+    * Sets the timezone for date/time/timestamp conversions.
+    */
+  def setTimeZoneId(zoneId: ZoneId): Unit = {
```

Comment:
Don't think we should add to this class, we are about to remove this TableConfig

Reply:
Yeah, I'll keep it be consistent with common TableConfig.